x/clerk/keeper/keeper.go

```diff
@@ -0,0 +1,227 @@
+package keeper
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+	"errors"
+
+	"github.com/tendermint/tendermint/libs/log"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/maticnetwork/heimdall/x/clerk/types"
+)
+
+var (
+	StateRecordPrefixKey = []byte{0x11} // prefix key for when storing state
+
+	// DefaultValue default value
+	DefaultValue = []byte{0x01}
```

Comment:
default value for?

Reply:
it is used for setting value for a record sequence

Reply:
Let's update the comment to make it clear